tests/diff/float_values_test.go

```diff
@@ -29,50 +29,45 @@ func TestFloatValues(t *testing.T) {
 	t.Parallel()
 
 	ctx, db := common.Setup(t)
-
-	t.Run("Insert", func(t *testing.T) {
```

Comment:
This subtest is not needed as we only test inserts anyway.

Reply:
Well, we should also do validation on updates, findAndModify, etc. Let's keep and add a stub for updates